alternator/executor.cc

```diff
@@ -105,6 +105,21 @@ static const column_definition& attrs_column(const schema& schema) {
     return *cdef;
 }
 
+
+static std::shared_ptr<stats> get_stats_from_schema(service::storage_proxy& sp, const schema& schema) {
+    try {
+        replica::table& table = sp.local_db().find_column_family(schema.id());
+        if (! table.get_stats().alternator_stats) {
+            table.get_stats().alternator_stats = std::make_shared<table_stats>(schema.ks_name(), schema.cf_name());
```

Comment:
I'm confused what this does - is it for the first time the table is ever operated on, and we didn't allocate stats for it?
Does it mean that if an Alternator exists but nobody does ANY operations on it, it won't get statistics at all - not even zeros?
I wonder if it makes sense to allocate those alternator_stats when creating the "table" object (if it's an Alternator table), even before any Alternator operation gets called on it. But maybe we will also need this for system tables, that Alternator can read?

Anyway, were is these metrics "registered"? Here? So doesn't this still have a bug when you delete and create a table in parallel?

Reply:
Keep in mind that we register the metrics once per shard, per node, but it's not only when the table is created, it would also happen after the restart, or on a node that didn't get the create table instruction.

Delete and create are not using the per-table stats, so there is no issue with them.

Reply:
Ok. I agree I can't think of a scenario where this won't work. In any case, even if we have such a problem it's not a big disaster, at worst we just get an error.